Makefile

```diff
@@ -83,55 +86,78 @@ bin:         ## Build all go binaries from cmd/.
              ##
 bin: $(GOCMDS)
 
+.PHONY: regen
 regen:       ## Refresh autogenerated files and reformat code.
-             ## Use DRY_RUN=1 to only validate without generating changes.
+             ## Use CHECK=1 to only validate clean repo after run.
              ##
 regen:
-ifeq ($(DRY_RUN), 1)
+ifeq ($(CHECK), 1)
 	$(call docker_build, -f ./hack/Dockerfile --target hermetic -t gmp/hermetic \
 		--build-arg RUNCMD='./hack/presubmit.sh all diff' .)
 else
 	$(call docker_build, -f ./hack/Dockerfile --target sync -o . -t gmp/sync \
-		--build-arg RUNCMD='./hack/presubmit.sh' .)
+		--build-arg RUNCMD='./hack/presubmit.sh all' .)
 	rm -rf vendor && mv vendor.tmp vendor
 endif
 
-test:        ## Run all tests. Setting NO_DOCKER=1 writes real data to GCM API under PROJECT_ID environment variable.
-             ## Use GMP_CLUSTER, GMP_LOCATION to specify timeseries labels.
+.PHONY: test
+test:        ## Run unit tests. Setting NO_DOCKER=1 runs test on host machine.
              ##
-	@echo ">> running tests"
+	@echo ">> running unit tests"
 ifeq ($(NO_DOCKER), 1)
-	kubectl apply -f manifests/setup.yaml
```

Comment:
Did you mean to delete applying the setup manifests to the cluster?

Reply:
Good catch, it really depends how others deploy the cluster and what's there. Will bring back for compatibility, someone might depend on those steps (operator access roles, setups being there). I can also try to verify this on my cluster... if I have time.